html/pfappserver/lib/pfappserver/Base/Form.pm

```diff
@@ -65,6 +65,11 @@ sub build_update_subfields {{
         element_class => ['input-mini'],
         element_attr => {'min' => '0'},
        },
+       'PSK' =>
```

Comment:
This field type would be better named PSKLength instead since its not actually the PSK that goes in there but its length